modules/monix/src/weaver/monixcompat/suites.scala

```diff
@@ -0,0 +1,24 @@
+package weaver
+package monixcompat
+
+import cats.effect.{ ConcurrentEffect, ContextShift, Resource, Timer }
+
+import monix.eval.Task
+import monix.eval.instances._
+import monix.execution.Scheduler
+
+trait BaseMonixSuite { self: ConcurrentEffectSuite[Task] =>
+  val ec                              = scala.concurrent.ExecutionContext.global
+  val scheduler                       = Scheduler(ec)
+  implicit def timer: Timer[Task]     = Task.timer(scheduler)
+  implicit def cs: ContextShift[Task] = Task.contextShift(scheduler)
+  implicit def effect: ConcurrentEffect[Task] =
+    new CatsConcurrentEffectForTask()(scheduler, Task.defaultOptions)
```

Comment:
It remains to see whether these defaults are sensible.

Reply:
I think it's okay